blueprints/addon/files/README.md

```diff
@@ -10,12 +10,15 @@ Installation
 ember install <%= addonName %>
 ```
 
-
 Usage
 ------------------------------------------------------------------------------
 
 [Longer description of how to use the addon in apps.]
 
+Compatibility
```

Comment:
I would propose to put this section above "Usage" (possible even above "Install") because "Usage" could get quite long 🤔

Reply:
Yes, I think that's a good point.